spec/models/user_spec.rb

```diff
@@ -14,4 +14,13 @@
       expect(user).to_not be_valid
     end
   end
+
+  describe 'balance' do
+    it 'calculate' do
+      user = create(:user)
+      transaction1 = create(:transaction, user_id: user.id)
+      transaction2 = create(:transaction, user_id: user.id)
```

Comment:
@marderer let's not overload `it` block with test preparation. We have `describe` and `before` block for that 😉 
What do you think?